src/receive_imf.rs

```diff
@@ -758,7 +758,8 @@ async fn add_parts(
                         // message is `MessageState::InNoticed`, which means that all following
                         // messages are sorted under it.
                         let sort_timestamp =
-                            calc_sort_timestamp(context, sent_timestamp, chat_id, true).await?;
+                            calc_sort_timestamp(context, sent_timestamp, chat_id, true, incoming)
```

Comment:
`incoming == true` here, btw. Also i'd suggest to comment passed bool values or use enums instead, bool flags are not well-read

Reply:
> `incoming == true` here, btw

Is this just something you noticed, or you suggest replacing it with `true`?

> Also i'd suggest to comment passed bool values or use enums instead, bool flags are not well-read

For me, it'd be as quick to jump to the method definition and seeing what the parameter does as finding the comment that belongs to it and reading it. An enum would be nicer to read, but seemed overkill for me for a function that's used just two times.

That being said, it's not important to me, so I can also change it.

Reply:
I'd suggest to replace it with smth like `/* incoming = */ true`, otherwise it can be read by mistake as a common code for incoming and outgoing messages. The same for `always_sort_to_bottom`, no long comments are needed. I wonder why Rust has named struct members initialisation, but no named function args.
Anyway it's minor, so up to you